Remove parameter inputEvent from fmi3EnterEventMode() (#1067)
modelica/fmi-standard: docs/3_2_model_exchange_api.adoc

> The followings arguments can be set to `fmi3True` to inform the FMU why *Event Mode* was entered.

Reviewer: ```suggestion
The followings function arguments have to be given to inform the FMU why *Event Mode* was entered.
```

> _[These arguments are not mutually exclusive and may all be `fmi3False`, e.g. for an <<input event>>.]_

Reviewer: I would not mention fmi3False here at all anymore because the rootsFound argument is not a boolean but an int array.
```suggestion
_[These arguments are not mutually exclusive.]_
```
